Sources/Controllers/MessagesViewController.swift

```diff
@@ -328,3 +366,35 @@ fileprivate extension MessagesViewController {
         return 0
     }
 }
+
+// MARK: - Menu Handling
+extension MessagesViewController {
+    /// Add observer for `UIMenuControllerWillShowMenu` notification
+    fileprivate func addMenuControllerObservers() {
+        NotificationCenter.default.addObserver(self, selector: #selector(MessagesViewController.menuControllerWillShow(aNotification:)), name: NSNotification.Name.UIMenuControllerWillShowMenu, object: nil)
```

Comment:
Can we use the shorthand version of the notification names please `.UIMenuControllerWillShowMenu`?